CrazyCanvas/Source/EventHandlers/AudioEffectHandler.cpp

```diff
@@ -22,6 +22,6 @@ void AudioEffectHandler::Init()
 bool AudioEffectHandler::OnProjectileHit(const ProjectileHitEvent& projectileHitEvent)
 {
 	using namespace LambdaEngine;
-	m_pProjectileHitSound->PlayOnceAt(projectileHitEvent.CollisionInfo0.Position);
+	//m_pProjectileHitSound->PlayOnceAt(projectileHitEvent.CollisionInfo0.Position);
```

Comment:
Thank you!

Reply:
Now the sound comes when you fire instead 😉